### DISCUSSION TITLE
PrimeNG Verona Template Angular 18 Support

### DISCUSSION THREAD

**Discussion by puckowski** [primefaces/community#2277] [PrimeNG Templates]
Are there any plans to upgrade the Verona template to be based on Angular 18? I can not find an existing discussion related to Angular 18 and Verona. 

My company purchased a Verona license because there were no plans to update Serenity to Angular 17 (https://github.com/orgs/primefaces/discussions/1802). 

Last I checked, the latest Verona template was built on Angular 17.

We are hoping to update all of our dependencies to versions built on/supporting Angular 18.

Only semi-related discussion I can find is https://github.com/orgs/primefaces/discussions/2120 which seems unanswered.

**Comment by giosueDelgado** [comment]
Hi @puckowski we also are waiting the update to Angular 18 so we hope the team can proceed in the next days.

I add a comment also to follow the thread.

@ZanchiStefano @HeorhiN

**Comment by enunez8** [comment]
alguna noticia de cuando podriamos descargar verona  v18? (Any news on when we could download Verona v18?)
